ubeshi/Wolf-in-sheep-s-clothing
# Implement culprit hints

## How to Test
1. Once in the investigation phase, move to a room with a character (excluding the horse)
2. Open the culprits menu to confirm no hints are shown at this point
3. Close the culprits menu and click on the character in the room
4. Once the conversation is over, open the culprits menu to confirm that a new hint is shown
5. Complete all conversations until the quest begins
6. Open the culprits menu to confirm that only two hints are shown for the character
7. Complete the quest
8. Confirm that all hints are showing in the culprits menu

## Notes
* Some timelines provide more than one hint. Each timeline should only provide one hint. Additionally, Each culprit should have four corresponding timelines (hint 1 → hint 2 → quest start → quest end + hint 3).
  * This has been updated in this PR
* The third dog hint has been updated based on the script
